File: JavaScript/1150.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

let x, z, soma = 0;
let cont = 0;

x = parseInt(lines[0])

let i = 1;

while(true){
    if(z > x){
        break
    }
    z = parseInt(lines[i])
    i++;
}

for(i = x; soma < z; i++){
    soma += i;
    cont++;
}

console.log(cont);

File: C++/1150.cpp
#include<iostream>

using namespace std;

int main(){
    int x, z, soma = 0;
    int cont = 0, i;

    cin >> x;

    while(true){
        cin >> z;

        if(z > x){
            break;
        }
    }

    for(i = x; soma < z; i++){
        soma += i;
        cont++;
    }
    
    cout << cont << endl;

    return 0;
}